packages/fxa-admin-server/nest-cli.json

```diff
@@ -15,5 +15,5 @@
       }
     ]
   },
-  "entryFile": "src/packages/admin-server/main.js"
+  "entryFile": "packages/fxa-admin-server/src/main.js"
```

Comment:
@dschom is this possibly related to whatever's happening with strict mode or was it still happening without this change? What's the error? Nothing else stands out. 

Your commit message says "Fixes bug where StrictMode was stopping lazy queries from executing" - should we remove that? I this is OK to land especially since it's admin-panel, but yeah let's investigate in another ticket. 👍